src/HomePage/HomePage.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './HomePage.css';
+
+class HomePage extends Component {
+    constructor(props) {
+        super();
+        this.toggleBack = this.toggleBack.bind(this);
+        this.state = { DisplayLogin: props.DisplayLogin };
+    }
+
+    toggleBack() {
+        this.setState({ DisplayLogin: !this.state.DisplayLogin });
+    }
+
+    render() {
+        return (
+            <div>
+                <div>
+                    <div className="upcoming-meetings-title">
+                        Upcoming Meetings (4)
+                    </div>
+                    <div className="upcoming-meetings-contents">
+                        <div className="upcoming-meetings-container">
+                            <table width="960">
```

Comment:
Could the width be set in the css file for these tables for consistency?
Also could you add `<thead>` and `<tbody>` tags to the tables, otherwise it causes this error in the console:
`Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody> to your code to match the DOM tree generated by the browser.`